apps/frontend/src/hooks/algolia.ts

```diff
@@ -3,7 +3,16 @@ import algoliasearch from 'algoliasearch/lite';
 import { useCurrentUser } from '@asap-hub/react-context';
 import { User } from '@asap-hub/auth';
 
-import { ALGOLIA_APP_ID, ALGOLIA_INDEX } from '../config';
+import {
+  ALGOLIA_APP_ID,
+  ALGOLIA_INDEX,
+  ALGOLIA_INDEX_REPLICA,
+} from '../config';
+
+export const INDEX = {
+  primary: ALGOLIA_INDEX,
+  'desc(addedDate)': ALGOLIA_INDEX_REPLICA,
```

Comment:
I don't like the name of this field: "desc(addedDate)".
I have never seen such convention,